src/javascript/app/pages/cashier/cashier.js

```diff
@@ -43,6 +43,12 @@ const Cashier = (() => {
         });
     };
 
+    const showP2PNote = (currency) => {
+        const is_currency_allowed = currency === 'USD';
```

Comment:
we should check the same logic as https://github.com/binary-com/deriv-app/pull/642/files#diff-bfde5894a635c6c5dc443bf9ed1f678fR155

just USD is not enough, client should also be either an agent or have at least one buy/sell offer available.